packages/amplify_authenticator/test/ui/theme_test.dart

```diff
@@ -0,0 +1,202 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Note: I renamed this file because the name didn't make sense anymore. New name + all the changes are making this look like a new file.